build-suggestions/README.md

```diff
@@ -1,4 +1,26 @@
 # build-suggestions
-This directory contains the build suggestions from OTA to ART. ART builds _primary metadata_ for each OpenShift release: set "Previous" value with all versions from `x.y` and `x.y-1` that are within those ranges. When scraping OpenShift releases, Cincinnati becomes an indirect consumer of the suggestions.
+This directory contains the build suggestions from OTA to ART. ART builds [primary metadata](https://github.com/openshift/cincinnati/blob/master/docs/design/openshift.md#update-image) for each OpenShift release: set the "previous" value with all versions from `x.y` and `x.y-1` that are within those ranges. When scraping OpenShift releases, Cincinnati becomes an indirect consumer of the suggestions.
 
 The file `build-suggestions/x.y.yaml` is initialized right after OpenShift repos cut the dev branch for `release-x.y` by running [release-open.sh](../hack/release-open.sh) and then creating a pull request like [cincinnati-graph-data#7239](https://github.com/openshift/cincinnati-graph-data/pull/7239).
+
+## semantics
+
+The filename `x.y.yaml` denotes that it contains the build suggestions for version `x.y`. For example, `build-suggestions/4.3.yaml`:
+
+```yaml
+default: # the architecture of the release for which the suggestions are. The suggestions under it are used if no other keys matches the architecture. It has to contains all the fields below.
+  minor_min: 4.2.21 # block all upgrade paths from 4.2.z to any 4.3 if z < 21
+  minor_block_list: [] # block all upgrade paths from 4.2.z to any 4.3 if 4.2.z is in the list
+  minor_max: 4.2.9999 # block all upgrade paths from 4.2.z to any 4.3 if z > 9999
```

Comment:
nit: possible clearer wording:

> of the previous minor version, 4.(y-1).z, 4.2.9999 is the maximum version to include. This excludes 4.2.(z>9999).  Using 9999 is a placeholder for "never actually exclude anything", because we will never maintain any 4.y branch long enough to have ten thousand patch releases.

Reply:
Updated (with "Using" removed and "ten thousand" -> "10,000").